couchdb/values.yaml

```diff
@@ -163,3 +163,21 @@ couchdbConfig:
 # This is used to generate FQDNs for peers when joining the CouchDB cluster.
 dns:
   clusterDomainSuffix: cluster.local
+
+# Adds a sidecar to expose CouchDB metrics as a prometheus endpoint
+# see https://github.com/gesellix/couchdb-prometheus-exporter/blob/master/README.md
+telemetryImage:
+  repository: gesellix/couchdb-prometheus-exporter
+  tag: latest
```

Comment:
Maintainer of the gesellix/couchdb-prometheus-exporter here :)

Would it be possible to use an explicit tag like `v27.2.0`, maybe using another config parameter to give users of this chart more flexibility? I currently don't provide any guarantees that `latest` points to an official release, so it might be some broken version for a short while. I'm open for feedback from the community, though.